### PR TITLE
Classic UI: icon for group depending on state

### PR DESCRIPTION
**Pull request by lolodomo**
Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Comment by lolodomo** [comment]
@resetnow : even if it is a fix for Classic UI, maybe you can have a look to this little code change ?

File: extensions/ui/org.eclipse.smarthome.ui.classic/snippets/group.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <li class="iGroup">
     <a href="#_%id%" onclick="AsyncLoad('%id%')">
-        <img src="../icon/%category%.%format%" width=29 height=29 />
+        <img src="../icon/%category%?format=%format%&state=%state%" width=29 height=29 />
```
Comment (kaikreuzer) [inline extensions/ui/org.eclipse.smarthome.ui.classic/snippets/group.html]: Does this correctly work if there is no state (which is probably the normal situation for Groups)?

Comment (lolodomo) [inline extensions/ui/org.eclipse.smarthome.ui.classic/snippets/group.html]: Normally "NULL" should be used and it should work but this should be tested.

Comment (lolodomo) [inline extensions/ui/org.eclipse.smarthome.ui.classic/snippets/group.html]: As it is working well in Basic UI with such URL, I am very confident.

Comment (kaikreuzer) [inline extensions/ui/org.eclipse.smarthome.ui.classic/snippets/group.html]: ok.

**Pull request merged**